src/plugins/data/common/es_query/filters/phrases_filter.ts

```diff
@@ -60,7 +55,7 @@ export const buildPhrasesFilter = (
   }
 
   return {
-    meta: { index, type, key, value, params },
```

Comment:
We actually weren't using the field formatter correctly here, showing the non-formatted values on the filter.
I assume this is a regression from the new index patterns changes.